### ISSUE TITLE
[Accessibility]A11y_ Azure Data Studio_ NotebookQueryTable_ScreenReader:Screen reader is announcing the unnecessary  information after "Toggle More" menu item .

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5614]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
NVDA Version: 2019.1
JAWS Version: 2019

**Additional Details:**
Tools Name: NVDA,JAWS
MAS Number :MAS1.3.1

**Steps to Reproduce** :
1.	Install the app Azure Data Studio.
2.	Connect to the Database->Go to file->New Notebook.
3.	Click on “Code” button and write query in the box for example SELECT * FROM sys.traces;
4.	Select query->Click on the Run command .
5.	Navigate to the “Toggle More” menu item using up and down arrows.
6.	Observe that Screen reader is  announcing unnecessary information after “Toggle More” menu item.

**Actual:
Issue 1:**
Navigate to the “Toggle More” menu item using up and down arrows, Screen reader is announcing the “toggle More” after that it also announces “menu submenu row 1 column 1”.
Same issue observed with JAWS.

**Issue 2:**
After clicking on “Toggle More” button, Screen reader doesn’t announce whether menu item list is expanded or not .

**Expected: 
Issue 1:**
Screen Reader should announce “Toggle More ”menu item.
**Issue 2:**
Screen reader should announce whether menu item list is expanded or not.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User with disabilities find it difficult to understand whether menu item list is expanded or not..

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by AccessibilityTestingTeam-TCS** [comment]
I checked this issue on following environment 
Version: 1.9.0-insider (user setup)
Commit: ad805b993c95e65e600921145c4f8d7602205ab7
Date: 2019-07-11T07:43:05.264Z
VS Code 1.34.0
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
and the issue is still reproduced.

**Comment by AccessibilityTestingTeam-TCS** [comment]
I checked this issue on following environment:
Version: 1.10.0-insider (user setup)
Commit: f720ec6
Date: 2019-07-15T07:47:15.691Z
VS Code 1.34.0
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
and the First issue is Fixed.
But second issue is still reprodued.
[5614_[Accessibility]A11y_ AzureDataStudio_ NotebookQueryTable_ScreenReader_Reopened.pptx](https://github.com/microsoft/azuredatastudio/files/3405183/5614_.Accessibility.A11y_.AzureDataStudio_.NotebookQueryTable_ScreenReader_Reopened.pptx)
